LEGO1/lego/legoomni/include/legoracespecial.h

```diff
@@ -35,14 +35,6 @@ class LegoCarRaceActor : public virtual LegoRaceActor {
 		return !strcmp(p_name, LegoCarRaceActor::ClassName()) || LegoRaceActor::IsA(p_name);
 	}
 
-	inline MxU32 VTable0x6c(
```

Comment:
I don't think we can delete this code. I'd just remove the `inline` keyword. That should solve the warning?